fixtures/rabota_fixtures.py

```diff
@@ -0,0 +1,98 @@
+import re
+
+import pytest
+
+from clients.http_client import HTTPClient
+from models.parser_jobs_tut_by import RabotaByParser
+
+url = 'https://rabota.by/search/' \
+      'vacancy?L_is_autosearch=false&area=16&clusters=true&enable_snippets=true&text=python&page=0'
+HEADERS = {
+    'user-agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/webkit-version (KHTML, like Gecko) '
+                  'Silk / browser - version like Chrome / chrome - version Safari / webkit - version'}
+
+
+@pytest.fixture
+def get_response():
+    return HTTPClient.get(url, headers=HEADERS)
+
+
+@pytest.fixture
+def get_response_from_wrong_url():
+    wrong_url = 'https://rabota.by/search/vacncy?clusters=true&' \
+                'area=16&enable_snippets=true&salary=&st=searchVacancy&text=python'
+    return HTTPClient.get(wrong_url, headers=HEADERS)
+
+
+@pytest.fixture
+def get_query_shotgun():
+    url_shotgun = 'https://rabota.by/search/' \
+                  'vacancy?L_is_autosearch=false&area=16&clusters=true&enable_snippets=true&text=shotgun'
+    return HTTPClient.get(url_shotgun, headers=HEADERS)
+
+
+@pytest.fixture
+def get_description_for_first_vacancy():
+    response = HTTPClient.get(url, headers=HEADERS).text
+    lxml_text = RabotaByParser.get_lxml(response)
+    list_id = RabotaByParser.get_list_of_vacancy_id(lxml_text)
+    first_vacancy_id = list_id[0]
+    response_for_first_vacancy = HTTPClient.get(f'https://rabota.by/vacancy/{first_vacancy_id}?query=python',
+                                                headers=HEADERS).text
+    description = RabotaByParser.find_vacancy_description(response_for_first_vacancy)
+    return description.text.lower()
```

Comment:
lxml_text = RabotaByParser.get_lxml(response)
    list_id = RabotaByParser.get_list_of_vacancy_id(lxml_text)
    first_vacancy_id = list_id[0]
    response_for_first_vacancy = HTTPClient.get(f'https://rabota.by/vacancy/{first_vacancy_id}?query=python',
                                                headers=HEADERS).text
    description = RabotaByParser.find_vacancy_description(response_for_first_vacancy)
    return description.text.lower()


all this logic should be implemented inside Parser class

Reply:
Done